Lib/fontTools/varLib/varStore.py

```diff
@@ -120,6 +120,15 @@ def __getitem__(self, varidx):
 			delta += d * s
 		return delta
 
+	def applyScalars(self, varDataIndex, deltas):
```

Comment:
Please rename to `interpolateFromDeltas`, to be consistent with `VariationModel`.

Reply:
Done.